mozilla-ai/lumigator
# Replace localstack with minio

# What's changing

The localstack service is replaced with minio for local deployments.

Closes https://github.com/mozilla-ai/lumigator/issues/487

# How to test it

Start lumigator (e.g. using `make local-up`) and check that minio starts instead of localstack.

All automated tests should work as usual.

# Additional notes for reviewers

N/A

# I already...

  - No DB update is needed.
